src/main/native/org_bblfsh_client_v2_libuast_Libuast.cc

```diff
@@ -156,6 +149,28 @@ JNIEXPORT jobject JNICALL Java_org_bblfsh_client_v2_Context_root(JNIEnv *env,
   return p->RootNode();
 }
 
+JNIEXPORT jobject JNICALL Java_org_bblfsh_client_v2_Context_encode(
+    JNIEnv *env, jobject self, jobject node) {
+  UastFormat fmt = UAST_BINARY;  // TODO(bzz): make it argument & enum
+
+  ContextExt *p = getHandle<ContextExt>(env, self, "nativeContext");
+  return p->Encode(node, fmt);
+  return nullptr;
+}
+
+JNIEXPORT void JNICALL Java_org_bblfsh_client_v2_Context_dispose(JNIEnv *env,
+                                                                 jobject self) {
+  ContextExt *p = getHandle<ContextExt>(env, self, "nativeContext");
+  setHandle<ContextExt>(env, self, 0, "nativeContext");
```

Comment:
Maybe move `"nativeContext"` to a constant?

Reply:
addressed in 0d76c98dbdf5ec8db5dedb43dda35a26d5f59c13